code/controllers/subsystem.dm

```diff
@@ -58,9 +58,9 @@
 		state = SS_PAUSED
 		queued_time = QT
 
-//previously, this would have been named 'process()' but that name is used everywhere for different things!
-//fire() seems more suitable. This is the procedure that gets called every 'wait' deciseconds.
-//Sleeping in here prevents future fires until returned.
+///previously, this would have been named 'process()' but that name is used everywhere for different things!
+///fire() seems more suitable. This is the procedure that gets called every 'wait' deciseconds.
+///Sleeping in here prevents future fires until returned..
```

Comment:
```suggestion
/*previously, this would have been named 'process()' but that name is used everywhere for different things!
   fire() seems more suitable. This is the procedure that gets called every 'wait' deciseconds.
   Sleeping in here prevents future fires until returned..
*/
```